hls4ml/writer/quartus_writer.py

```diff
@@ -814,6 +875,9 @@ def write_hls(self, model):
         self.write_bridge(model)
         self.write_build_script(model)
         self.write_nnet_utils(model)
+        self.write_activation_lstm(model)
```

Comment:
It is needed to have a condition, if the activation is not lstm we don't have to do this function